Issue while trying to commit

```
/Users/wojciech.maj/Library/pnpm/store/v3/tmp/dlx-7170:
 ERR_PNPM_FETCH_404  GET https://registry.npmjs.org/--no-install: Not Found - 404

This error happened while installing a direct dependency of /Users/wojciech.maj/Library/pnpm/store/v3/tmp/dlx-7170

--no-install is not in the npm registry, or you have no permission to fetch it.

An authorization header was used: Bearer npm_[hidden]
```

Likely some issue with lint-staged as commiting with `-n` flag works just fine for me. 

I'm using pnpm 7.14.0.

--- a/package.json
+++ b/package.json
@@ -17,7 +17,8 @@
     "generate:supersprite": "node ./scripts/generateSuperSVG.mjs",
     "optimize": "node ./scripts/optimizeSvgs.mjs",
     "addtags": "node ./scripts/addMissingKeysToTags.mjs",
-    "generate:changelog": "node ./scripts/generateChangelog.mjs"
+    "generate:changelog": "node ./scripts/generateChangelog.mjs",
+    "postinstall": "husky install"
   },
   "devDependencies": {
     "@atomico/rollup-plugin-sizes": "^1.1.4",
@@ -39,7 +40,7 @@
     "eslint-config-prettier": "^2.10.0",
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-prettier": "^2.7.0",
-    "husky": "^4.3.8",
+    "husky": "^8.0.0",
     "jest": "^28.1.3",
     "lint-staged": "^10.5.4",
     "minimist": "^1.2.6",

